WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Dynamic/DynamicDashboardCard.swift

```diff
@@ -0,0 +1,143 @@
+import SwiftUI
+import DesignSystem
+
+struct DynamicDashboardCard: View {
+    private enum Constants {
+        static let featureImageHeight: CGFloat = 151
+        static let rowImageDiameter: CGFloat = 48
+    }
+
+    struct Input {
+        struct Row: Identifiable {
+            let id: UUID = UUID()
+
+            let title: String?
+            let description: String?
+            let imageURL: URL?
+        }
+
+        struct Action {
+            let title: String?
+            let callback: (() -> Void)
+        }
+
+        let featureImageURL: URL?
+        let rows: [Row]
+        let action: Action?
+    }
+
+    private let input: Input
+
+    init(input: Input) {
+        self.input = input
+    }
+
+    var body: some View {
+        VStack(spacing: Length.Padding.single) {
+            featureImage
+            rowsVStack
+            actionHStack
+        }
+    }
+
+    @ViewBuilder
+    var featureImage: some View {
+        if let featureImageURL = input.featureImageURL {
+            AsyncImage(url: featureImageURL)
```

Comment:
@alpavanoglu Should we adjust the aspect ratio for this image and all other "AsyncImages" or the default one looks good?

Reply:
I am not sure really. It might be easier to test it with real images. Do we already have a working URL or image already?

Reply:
@alpavanoglu Maybe we can test using the Domain Management card @osullivanchris designed. We can export the images from Figma and upload them somewhere. Or you can use a fake image generator, like this one https://pickadummy.com/

Reply:
That website looks very useful actually. Thanks!